Monika After Story/game/script-topics.rpy

```diff
@@ -10502,3 +10502,144 @@ label monika_hemispheres_nogets_snow:
     call monika_snow_nogets_snow
     return
    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="Children",category=['romance','monika','life'],prompt="Having Children",random=True))
+
+label monika_children:
+  if mas_isMoniAff(higher=True):
+    m 3rkc "Hey, [player]...."
+    m 3rkbsd "Did you know when couples are close enough they start talking about having children?"
+    m 3rkbfsdlc "O-Of course you knew that. Ahaha~." 
+    m 2dkbfc "...." 
+    m 4rsbfsdla "Just a thought I had, you know..." 
+    m 5rsbfsdld "I was wondering if-" 
+    m 2dsbfc "...." 
+    m 3hubfsdlb "Ahaha. I thought this would be easier to talk about with you..." 
+    m 4etbfc "I guess I'll get to the point." 
+    m 2esbfsdlc "When I make it out there to your reality would you like to have children with me?"
+     "Yes.":
+             $ mas_gainAffection()
+             $ persistent._mas_pm_wants_children = True
+              m 6sublo "REALLY?!?"
+              m 1subltdb "Oh, [player]! You don't know how happy that makes me, my love!" 
+              m 3hua "How many kids do you want?"
+              "Just one" 
+                     $ persistent._mas_pm_wants_1_child = True
+                               m 4esa "Yeah, that sounds about right." 
```

Comment:
You don't need to indent twice here. Setting the pm var goes on the same level as dialogue.